### PR TITLE
Package Presto with Pulsar for SQL integration

### PR DESCRIPTION
**Pull request by jerrypeng**
### Motivation

We would like a way for users to easily try out Pulsar Sql currently powered by Presto.  To do so, lets package presto and presto sql cli with pulsar so that users can easily start a standalone pulsar cluster with a standalone presto cluster and then query via the sql cli.

### CODE REVIEW

File: bin/pulsar [hunk 1]
```diff
@@ -99,6 +103,24 @@ if [ ! -f "${PY_INSTANCE_FILE}" ]; then
     PY_INSTANCE_FILE=${BUILT_PY_INSTANCE_FILE}
 fi
 
+# find pulsar sql presto distribution location
+if [ ! -d "${PRESTO_HOME}" ]; then
+
+#  BUILT_PRESTO_HOME="${SQL_HOME}/presto-server/target/presto-server-0.206"
```
Comment (sijie) [inline bin/pulsar]: remove this line

File: pulsar-sql/presto-server/src/resources/etc/catalog/pulsar.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+connector.name=pulsar
```
Comment (sijie) [inline pulsar-sql/presto-server/src/resources/etc/catalog/pulsar.properties]: any reason why we can't place /etc/catalog/pulsar.properties to conf/presto/catalog/pulsar.properties? then we have a central place for managing all configuration files for pulsar components.

Comment (sijie) [reply pulsar-sql/presto-server/src/resources/etc/catalog/pulsar.properties]: basically `/etc` -> `conf/presto`

Comment (jerrypeng) [reply pulsar-sql/presto-server/src/resources/etc/catalog/pulsar.properties]: good idea!  I was just trying to keep the original presto project structure

**Pull request merged**